class AlreadyExistError(Exception):
    pass


class NoRowsFoundError(Exception):
    pass
